ext/reflection/php_reflection.c

```diff
@@ -5332,7 +5332,7 @@ ZEND_METHOD(reflection_property, __construct)
 	}
 
 	ZVAL_STR_COPY(reflection_prop_name(object), name);
-	if (dynam_prop == 0) {
+	if (property_info && dynam_prop == 0) {
```

Comment:
I am not an expert but we already checked if property_info == NULL on line 5307.

For sure makes the code "stronger" but I dont know if is needed.

The same kind of things happened in other places too.